components/network/openvpn/files/openvpn.xml

```diff
@@ -12,30 +14,51 @@
             <service_fmri value="svc:/system/filesystem/local"/>
         </dependency>
 
-        <method_context>
+        <method_context project=':default' resource_pool=':default' working_directory='/etc/openvpn'>
```

Comment:
project and resource_pool are something we shouldn't user unless we have support for it in IPS. As for working directory, won't it cause the issue, if people use different location for config file?